crates/common/src/lib.rs

```diff
@@ -560,6 +560,27 @@ macros::starkhash::common_newtype!(
 macros::fmt::thin_display!(StarknetBlockNumber);
 macros::fmt::thin_display!(StarknetBlockTimestamp);
 
+#[derive(Clone, Debug, PartialEq)]
+pub enum AllowedOrigins {
+    Any,
+    List(Vec<String>),
+}
+
+impl<S> From<S> for AllowedOrigins
```

Comment:
This is only used to save typing in tests, so maybe `#[cfg(any(test, feature = "test-utils"))]` ?